libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -612,21 +612,70 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
     display: flex;
     flex: 1 0 auto;
 
+    @mixin icon-state {
+        &.disabled {
+            color: $gd-color-disabled;
+        }
+
+        &.is-normal {
+            color: $gd-color-link;
+        }
+    }
+
+    @mixin hover-effect {
+        &:hover {
+            z-index: 100;
+            box-shadow: 0 0 0 1px $gd-color-highlight;
+
+            @include icon-state;
+        }
+    }
+
     .icon-sources {
         border-bottom-right-radius: 0;
         border-top-right-radius: 0;
 
-        &:hover {
-            border-color: $gd-color-highlight;
+        &.is-normal::before {
+            margin-top: 2px;
+            margin-left: 2px;
+            background: url("~@gooddata/sdk-ui-kit/esm/assets/data_source_normal_state.svg") no-repeat;
+        }
+
+        &.is-active::before {
+            background: url("~@gooddata/sdk-ui-kit/esm/assets/data_source_active_state.svg") no-repeat;
+        }
+
+        &.disabled::before {
+            background: url("~@gooddata/sdk-ui-kit/esm/assets/data_source_disabled_state.svg") no-repeat;
```

Comment:
This url looks weird

Reply:
https://github.com/gooddata/gooddata-ui-sdk/commit/30747e2df8d0357bc7971fea54caaa7abfc2e9e7 i just follow this PR

Reply:
ok